### PR TITLE
Changelog 0.68.0

### PR DESCRIPTION
**Pull request by lewri**
Adds changelog ready for next release.

NB: As we're now planning to use the revision number more strictly, we should probably also start adding the .0

The following text below is intended to go on the release post at the top of the changelog:

**This version comes with two experimental features you can help us with by giving feedback on! These will be off by default.**

### CODE REVIEW

**Comment by lewri** [comment]
If #2597 is merged I'll add it on later.

**Comment by MandrakeHorse** [comment]
Please can you add #2435 to the Bug Fixes section? I don't think it mentions that we fixed a missing sound when marking contagious patients for vaccination.

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,94 @@
...
+  playing fullscreen at non-native resolutions
+* Games should no longer crash irrecoverably because you have a 4k monitor :) 
+* The game will now exit to the main menu cleanly if a problem occurred trying
+  to load a new level or map
+* Staff who have left the hospital can no longer ask for a raise
+* Fixed a bug where some staff may have no initial before surname
+* Patients can no longer litter outdoors
+* Fixed a bug where an unreachable reception desk could cause a crash
+* The Computer and Atom Analyser now make button sounds as originally intended
+* Fixes a rare bug where edges of map tiles for parcels could cause unintended
```
Comment (lewri) [inline changelog.txt:66]:
```suggestion
* The Computer and Atom Analyser now make button sounds as originally intended
* The mark for vaccination action now makes a sound as originally intended
* Fixes a rare bug where edges of map tiles for parcels could cause unintended
```

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,95 @@
+-------------------------------------------------------------------------------
+CorsixTH 0.68.0 - released xxxx 2024
+-------------------------------------------------------------------------------
+# New Features/Enhancements
+* In game movies now play for users using TH Data directly from a .ISO file
```
Comment (Alberth289346) [inline changelog.txt:5]:
```suggestion
* In-game movies now play for users using TH Data directly from a .ISO file
```

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,95 @@
...
+  currently afford it
+* More cheats!
+* [Experimental] Right mouse panning can now be used instead of using the
+  middle mouse button. Enable it in the configuration file
+* [Experimental] We’re closer to fully implementing falling actions! Get a
+  sneak peek and get the chance to push people over by enabling it in the
+  configuration file/debug menu
+
+# Changes
+* The demo movie will no longer play and cause jumpscares at the main menu if
```
Comment (Alberth289346) [inline changelog.txt:23]:
```suggestion
* The demo movie will no longer play and cause jump-scares at the main menu if
```
not sure

File: changelog.txt [hunk 4]
```diff
@@ -1,3 +1,95 @@
...
+  instead of at the start
+* Errors with music playback will now attempt to provide more helpful
+  information in the console
+* Level briefings now show before the in-game tutorial
+* The first patient of a level will now arrive faster after opening the
+  hospital
+* The information dialog box is now more closely aligned with the original game
+* The tip of the day window should no longer be obscured in the main menu
+* The load/save windows now have better labelling
+* Unavailable languages are shown as disabled until you select a unicode font
```
Comment (Alberth289346) [inline changelog.txt:40]:
```suggestion
* Unavailable languages are shown as disabled until you select a Unicode font
```
?

File: changelog.txt [hunk 5]
```diff
@@ -1,3 +1,95 @@
...
+* Support is added to auto detect a Theme Hospital install via GOG Galaxy
```
Comment (Alberth289346) [inline changelog.txt:41]:
```suggestion
* Support is added to auto-detect a Theme Hospital install via GOG Galaxy
```

File: changelog.txt [hunk 6]
```diff
@@ -1,3 +1,95 @@
...
+* Fixes a rare bug where edges of map tiles for parcels could cause unintended
+  behaviour when purchasing plots
+* Active cheats will now persist across saves
+* Audio settings have better safeguards against no audio enabled/no background
+  music
+* Movies will no longer attempt to play audio when global audio is off
+* Config values using brackets (such as a custom music directory) will now work
+  properly
+* Custom campaigns menu now will use a scrollbar for long campaign descriptions
+* Continue game now properly targets files explicitly ending in the .sav format
```
Comment (Alberth289346) [inline changelog.txt:76]:
```suggestion
* Continue-game now properly targets files explicitly ending in the .sav format
```

**Comment by Alberth289346** [comment]
A number of entries use past tense ("fixed" or "implemented"), while others are current tense (eg "fixes").
Likely it makes sense to align that.

Also, do the sentences need a dot at the end?

**Comment by lewri** [comment]
I can amend the tense sure. It's common practice though for bullet lists not to end with a full stop as far as I've seen over the years.

File: changelog.txt (hunk 6 above)
Comment (lewri) [reply changelog.txt:76]: Might want to leave the dash out of this one to align with the button in the game

Comment (Alberth289346) [reply changelog.txt:76]: Your suggestion makes sense to me.

**Comment by Alberth289346** [comment]
> I can amend the tense sure. It's common practice though for bullet lists not to end with a full stop as far as I've seen over the years.

I work in another project where they do have full stops, perhaps it's a local custom :)
Either way is fine by me.

File: changelog.txt (hunk 6 above)
Comment (lewri) [reply changelog.txt:76]:
```suggestion
* Continue Game now properly targets files explicitly ending in the .sav format
```

File: changelog.txt [hunk 7]
```diff
@@ -1,3 +1,95 @@
...
+* Games should no longer crash irrecoverably because you have a 4k monitor :) 
+* The game will now exit to the main menu cleanly if a problem occurred trying
+  to load a new level or map
+* Staff who have left the hospital can no longer ask for a raise
+* Fixed a bug where some staff may have no initial before surname
+* Patients can no longer litter outdoors
+* Fixed a bug where an unreachable reception desk could cause a crash
+* The Computer and Atom Analyser now make button sounds as originally intended
+* The mark for vaccination action now makes a sound as originally intended
+* Fixes a rare bug where edges of map tiles for parcels could cause unintended
```
Comment (lewri) [inline changelog.txt:67]:
```suggestion
* Fixed a rare bug where edges of map tiles for parcels could cause unintended
```

File: changelog.txt [hunk 8]
```diff
@@ -1,3 +1,95 @@
...
+  behaviour when purchasing plots
+* Active cheats will now persist across saves
+* Audio settings have better safeguards against no audio enabled/no background
+  music
+* Movies will no longer attempt to play audio when global audio is off
+* Config values using brackets (such as a custom music directory) will now work
+  properly
+* Custom campaigns menu now will use a scrollbar for long campaign descriptions
+* Continue game now properly targets files explicitly ending in the .sav format
+* Fixes an instance where information boxes could load pink from older
```
Comment (lewri) [inline changelog.txt:77]:
```suggestion
* Fixed an instance where information boxes could load pink from older
```

File: changelog.txt [hunk 9]
```diff
@@ -1,3 +1,95 @@
...
+  savegames
+* Implemented a more permanent fix for the money bar being drawn incorrectly in
+  some CJK and Cyrllic languages
+* Fixes a crash on exit that could occur in some systems
```
Comment (lewri) [inline changelog.txt:81]:
```suggestion
* Fixed a crash on exit that could occur in some systems
```

File: changelog.txt (hunk 3 above)
Comment (lewri) [reply changelog.txt:23]: Annoyingly `jump scares`, `jump-scares`, and `jumpscares` are all acceptable in English, very helpful

**Comment by Alberth289346** [comment]
LGTM

**Pull request merged**